source/features/scheduled-and-manual-workflow-indicators.tsx

```diff
@@ -65,47 +65,56 @@ const getWorkflowsDetails = cache.function(async (): Promise<Record<string, Work
 	cacheKey: () => 'workflows:' + getRepo()!.nameWithOwner,
 });
 
-async function init(): Promise<false | void> {
+async function addIndicators(workflowListItem: HTMLAnchorElement): Promise<void> {
+	// Memoized above
 	const workflows = await getWorkflowsDetails();
 	if (!workflows) {
-		return false;
+		return; // Impossibru, for types only
 	}
 
-	const workflowsSidebar = await elementReady('.ActionList');
```

Comment:
Weirdest thing. This resolves but it's empty: `workflowsSidebar.innerHTML === ''` 🤷‍♂️ 

The data exists in the raw HTML so I'm not sure why elementReady is failing